docs/install.md

```diff
@@ -603,7 +697,7 @@ Select the following tabs for the installation description of your target.
 
         <font size="+2">1. Download and extract the DietPi direct write image</font>
 
-        Download the **BIOS direct write image** `DietPi_NativePC-BIOS-x86_64-Buster.7z` from [`dietpi.com`](https://dietpi.com/#download) and
+        Download the **BIOS direct write image** `DietPi_NativePC-BIOS-x86_64-Bullseye.7z` from [`dietpi.com`](https://dietpi.com/#download) and
```

Comment:
Probably we should remove the file name from here (and above cases) and replace it with what one actually sees at the linked download page:
```
Download the **Native PC (BIOS/CSM)** image from...
```
That way we do not need to remember updating those file names 😉.

Reply:
Ok. Done.